pkg/ccl/storageccl/export_storage.go

```diff
@@ -45,6 +50,19 @@ const (
 	AzureAccountNameParam = "AZURE_ACCOUNT_NAME"
 	// AzureAccountKeyParam is the query parameter for account_key in an azure URI.
 	AzureAccountKeyParam = "AZURE_ACCOUNT_KEY"
+
+	// AuthParam is the query parameter for the cluster settings named
+	// key in a URI.
+	AuthParam         = "AUTH"
+	authParamImplicit = "implicit"
+	authParamDefault  = "default"
+
+	cloudstoragePrefix           = "cloudstorage"
+	cloudstorageGS               = cloudstoragePrefix + ".gs"
+	cloudstorageDefault          = ".default"
+	cloudstorageGSDefault        = cloudstorageGS + cloudstorageDefault
+	cloudstorageGSJsonkey        = ".jsonkey"
```

Comment:
🚲🏡: arguably we don't need `jsonkey`, but it keep symmetry with the other providers where we'll have suffixes for `.key` and `.secret` or whatever.

Reply:
Changed to `.key`